Manuscript/Sasaki_and_Moreno_2023.Rmd

```diff
@@ -26,74 +26,83 @@ output:
+Long term increases in ocean temperatures can accelerate the metabolisms of marine organisms, due to the fact that most are ectothermic, linking the rates of their metabolic processes to their thermal environment @johnston2022. As a result, caloric requirements will increase, thus increasing consumption pressure on autotrophic organisms. Modeling of community-scale metabolism reveals that increases in ocean temperatures could result in increased NPP and decreased overall community biomass, while community structure shifted to increased heterotrophic biomass relative to autotrophic biomass @archibald2022 . although the precise changes in community structure were heavily dependent on region, the aforementioned changes could result in greater intra- and inter- specific competition, and overall result in the control of prey populations by apex predators @johnston2022. Furthermore, The reduction in dissolved oxygen associated with increased ocean temperatures further exacerbates resource limitations imposed by increased metabolic requirements and changes in marine community structure. On the other hand, MHWs threaten the stability of marine ecosystems by exposing marine organisms to temperatures beyond their optimum and into the lethal range. MHWs have grown in duration, intensity, and frequency over the last 4 decades @li2022, which in tandem with the aforementioned limitation of food resources, emphasizes the importance of studying the upper thermal limits of marine organisms, as represented by Critical Thermal Maxima (CT~max~).
+
+The interaction between starvation and thermal limits have been scantly studied in marine organisms, despite the threats posed by anthropogenic climate change. @huey2019 describes how simultaneous increases in environmental temperature and reduced food availability act to reduce the overall heat tolerance of an organism, in a process termed 'metabolic meltdown'. It is vital to understand how copepod thermal limits respond to changes in food availability in order to better model population dynamics in a changing climate.
```

Comment:
We need a broader survey of the literature on starvation and thermal limits. We can take a step back and look at how non-marine taxa are affected as well.